### PR TITLE
Adding tags (still ways to go)

### PR DESCRIPTION
**Pull request by blimpa**
@CPSTL, @Davidutro 

(Not happy with having most of the MIP10cX subproposals tagged as "oracles"; not happy with each having a unique tag either.)

### CODE REVIEW

**Comment by blimpa** [comment]
Long said I should be able to tag you guys as reviewers, but I can't figure out how. Anyway, @Davidutro, @CPSTL!

**Comment by Davidutro** [comment]
@blimpa

**Comment by blimpa** [comment]
> 
> 
> @blimpa

Thank you, David :) But there's no cog on my end :/

image

**Comment by Davidutro** [comment]
@LongForWisdom maybe its a permissions thing? ☝️

File: MIP1/mip1.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ MIP1 defines and describes Governance Paradigms and problem spaces.
 
 ## Paragraph Summary
 
-A Governance Paradigm is a complete and specific group of MIP Sets that solves an entire problem space. A problem space is defined as a list of issues that must be addressed in order for the Maker Protocol to be able to sustain itself and grow into the future Maker governance.
+A Governance Paradigm is a complete and specific group of MIP Sets that solve an entire problem space. A problem space is defined as a list of issues that must be addressed in order for the Maker Protocol to be able to sustain itself and grow into the future Maker governance.
```
Comment (Davidutro) [inline MIP1/mip1.md]: undid this change, it's grammatically correct.

File: MIP4/mip4.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ A process component which defines a method and template for the removal of an ac
 
 ## Motivation
 
-The motivation behind this proposal is that changing small details to MIPs should not require the original MIP to become obsolete or replaced, so this Process-MIP is needed to define and outline the process behind making these changes to MIPs once they have already been ratified and implemented. Additionally, this MIP defines the process for the removal of MIPs that are no longer useful.
+The motivation behind this proposal is that changing small details to MIPs should not require the original MIP to become obsolete or replaced, so this Process-MIP is needed to define and outline the process behind making these changes to MIPs once they have already been ratified and implemented. Additionally, this MIP defines the process for the removal of MIPs that are have become no longer useful.
```
Comment (Davidutro) [inline MIP4/mip4.md]: fixed wording

**Pull request merged**